modules/setting/setting.go

```diff
@@ -597,16 +598,16 @@ func NewContext() {
 		}
 	case "fcgi":
 		Protocol = FCGI
-	case "fcgi+unix":
-		Protocol = FCGIUnix
-		UnixSocketPermissionRaw := sec.Key("UNIX_SOCKET_PERMISSION").MustString("666")
```

Comment:
Why we removed these logic?

Reply:
I believe this just removes the duplication, the logic still exists below

Reply:
I don't think so. Following lines checked `700` but new code lost that.

Reply:
I refactored the code, but indeed the diff do not make it apparent. The case "unix" and "fcgi+unix" was the same, just a different Protocol value. So I merged them, and added the switch/case for the protocol and warning.